test/tools/evmoneRunner.h

```diff
@@ -0,0 +1,48 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include <deps/evmone/include/evmone/evmone.h>
+#include <string>
+
+namespace external
+{
+namespace evmone
+{
+class EvmOneVM : public evmc_context
```

Comment:
Is there any benefit in inheriting from `evmc_context` instead of having it as a member?

Reply:
Ah, I see there is! nevermind! :)

Reply:
Not that I can think of. @chfast : Do you foresee any problems if the evmone wrapper does not inherit from `evmc_context` rather makes it a member object?